app/controllers/export/document_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Export::DocumentController < Export::BaseController
+  self.responder = Api::Responder
+
+  def show
+    @document = Document.find_by(id: params[:id])
+    if @document
+      output = {
+                 "document": @document,
+                 "editions": []
+               }
+      @document.editions.each do |edition|
+        output[:editions].push(edition_associations(edition))
+      end
+      respond_with output
+    else
+      respond_with_not_found
+    end
+  end
+
+private
+
+  def respond_with_not_found
+    respond_with Hash.new, status: :not_found
+  end
+
+  def edition_associations(edition)
+    output = {
+               "edition": edition
+             }
+    if edition.class.methods.include? :reflect_on_all_associations
```

Comment:
Do we think there's much chance this would ever be false and we'd still want to proceed?

Reply:
My original plan was to iterate over the association's associations, which sometimes didn't have a `reflect_on_all_associations` method.  I'll remove this `if` statement since it's no longer needed (should've removed it before).